src/cpp/include/openvino/genai/text2image/txt2img_core.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "openvino/runtime/core.hpp"
+
+namespace ov {
+namespace genai {
+namespace txt2img_core {
+
+ov::Core singleton_core();
```

Comment:
Make it private. An attempt to keep it private is the reason we have this issue now, so stick to the original intention.